api-schema/src/main/java/org/stellar/anchor/api/platform/GetTransactionResponse.java

```diff
@@ -58,4 +58,13 @@ public class GetTransactionResponse {
 
   Customers customers;
   StellarId creator;
+
+  @SerializedName("to_account")
+  String toAccount;
+
+  @SerializedName("request_asset_code")
+  String requestAssetCode;
+
+  @SerializedName("request_asset_issuer")
+  String requestAssetIssuer;
```

Comment:
@JakeUrban 
1. `requestAssetCode` and `requestAssetIssuer` are necessary right now. AP does not set any assets (including asset of `amount_in` field). Maybe we should change it in scope of another PR?
2. Current schema is misleading. This field is always null for sep-24. There is no `creator`

Reply:
I'm a bit confused, is it currently setting `requestAssetCode` and `requestAssetIssuer`? Those fields aren't defined in the SEPs or the Anchor Platform APIs, so we should definitely update the platform. The same goes for SEP-24, I'm not sure why there is no `creator`. It sounds like we need to do some work on the anchor platform side.

I'd prefer if we made this changes to the Platform before merging this PR. There is no need to merge something that we know will be changed very soon.

Reply:
Yes it's setting `requestAssetCode` and `requestAssetIssuer` leaving SEP fields (`amount_in_asset`) to be set for external developer, you can check DB schema here
https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/java/org/stellar/anchor/platform/data/JdbcSep24Transaction.java
I think Anchor Platform API should be updated.
As for `creator`, it's not in SEP-24 transaction, so I'm not sure if it should be there

Reply:
SEP-31's APIs don't have a `creator` attribute either, but we need to provide the `creator` attribute because without it the business wouldn't know who is authenticated with the Platform API

Reply:
These fields are not in `schemas.yml`. It is a bit confusing trying to discuss the schema change in the implementation. Can we create a PR for the `schemas.yml`?

Reply:
I think they are in schemas.yml: https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/docs/03%20-%20Implementing%20the%20Anchor%20Server/Communication/schemas.yml#L290

Reply:
Hey are in schemas.yml, but they're part of the `Amount` object. Every `Amount` object has two fields:

```
    Amount:
      type: object
      required:
        - amount
        - asset
      properties:
        amount:
          type: string
        asset:
          type: string
```

Reply:
Hmm so the `asset` is required in the schema. For sep-24 it says asset is optional in the spec though...

Reply:
@Ifropc we can modify the schemas.yml to make asset optional.